esp-metadata/src/generate_cfg.rs

```diff
@@ -186,6 +186,20 @@ impl Config {
         }
     }
 
+    /// Create an empty configuration
+    pub fn empty() -> Self {
+        Self {
+            device: Device {
+                name: "NULL".to_owned(),
```

Comment:
Why `NULL` instead of just an empty string? Also why not implement `Default` instead?

Reply:
I'll change it to an empty string. I chose not to implement `Default` as it may imply there is a default chip, this is explicitly an empty configuration.